.github/CONTRIBUTING.md

```diff
@@ -12,6 +12,23 @@ Before you make a bigger pull request, please open an issue and make sure one of
 bug, please create a bug report issue and illustrate the bug with a minimal
 [reprex](https://www.tidyverse.org/help/#reprex).
 
+### Bug reports
+
+Please provide a code example of the bug you're encountering.
+A good code example should meet the following criteria:
+
+1. **Minimal**: Only include the essential code needed to illustrate the problem.
+Remove any extraneous code that isn't directly related to the issue at hand.
+2. **Complete**: Include everything that someone else would need to run the example,
+such as the full contents of the app.R file.
+3. **Reproducible**: Ensure that the problem actually appears in the example.
+Include information like the R and package version, operating system, and browser.
+4. **Readable**: Use individual Markdown code blocks for each file or code snippet.
```

Comment:
I suggest linking [this article](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax) for people who might not know markdown enough to write issue compliant to our criteria